js/app.js

```diff
@@ -101,14 +101,30 @@ function buildBlogSummary(blog, modularContent)
         }
     }
 
-    const body = firstOrDefaultValue(blog, "body") || "";
+    let body = firstOrDefaultValue(blog, "body") || "";
+    body = convertImagesToHyperlink(body);
 	blogSummary.querySelector(".summary-body").innerHTML = body;
 
     processContentSnippets(blogSummary, modularContent);
 	return blogSummary;
 }
 
 
+function convertImagesToHyperlink(blogBody) {
+    let wrapper = document.createElement("div");
```

Comment:
this could be const (other instances below).

Reply:
Yes, you're right